### ISSUE TITLE
password via stdin or file <feature request>

### ISSUE THREAD

**Issue by m-fonseca** [fdupoux/fsarchiver#120]
Hello,

There should be a way to to give a password to fsarchiver in an automated way without putting it on the cmdline for all to see.  Currently, -c only works directly with the password as a commadline argumet, or it prompts (which is no good for calling from a non interactive script unless you use something like expect to simulate user input.  

It would be great if -c could somehow take a file, or read the password from stdin.

**Comment by lexo-mfleuti** [comment]
Indeed, I am sturggling with this issue now for like an hour... it's very insecure to display the encryption password in the process list. Every system user can see it like that. And password prompting is not an option for automation.
